crates/cubecl-linalg/src/matmul/components/global/tensor_view/cyclic_loading.rs

```diff
@@ -0,0 +1,125 @@
+use crate::matmul::components::config::PlaneMapper;
+use crate::matmul::components::global::Config;
+use crate::matmul::components::stage::{
+    TilingOrder, TilingOrderConfig, XMajorTiling, YMajorTiling,
+};
+use crate::matmul::components::{Ident, MatrixLayout};
+use cubecl_core as cubecl;
+use cubecl_core::prelude::*;
+
+use super::base::TensorView;
+
+#[derive(CubeType, Clone, Copy)]
+/// Loads the content of all tiles in the tensor view using all planes,
+/// iterating with steps determined by the plane's dimension.
+pub struct CyclicLoading {}
+
+#[cube]
+impl PlaneMapper for CyclicLoading {
+    fn plane_id() -> u32 {
+        UNIT_POS_Y
+    }
+
+    fn plane_unit() -> u32 {
+        UNIT_POS_X
+    }
+}
```

Comment:
What is the plane mapper?

Reply:
Components must implement this trait when they use unit positions in the algorithm. It's a way to abstract the algorithms from unit positions. 

For now it's always plane_id=UNIT_POS_Y (which plane) and plane_unit=UNIT_POS_X (which unit within the plane)
So for  a cube dim of (32, 8, 1) plane_id is in 0..8 and plane_unit is in 0..32. 

It might be a bit overkill since every implementation of the plane mapper is the same for now, but the idea is that if we have plane specialization, for double buffering for instance, we can write the algorithm as if all planes contributed to the algorithm. For instance if we had planes 0..4 do something else and planes 4..8 performing the algorithm, then the plane mapper could map 4..8 to 0..4 so that the offset doesn't impact the algorithm. 

But that's future work anyway, for now it's really just abstracting unit positions in the algorithm.